posthog/hogql/constants.py

```diff
@@ -92,6 +92,8 @@ def get_breakdown_limit_for_context(limit_context: LimitContext) -> int:
 class HogQLQuerySettings(BaseModel):
     model_config = ConfigDict(extra="forbid")
     optimize_aggregation_in_order: Optional[bool] = None
+    use_query_cache: Optional[bool] = None
```

Comment:
Why use the ClickHouse built-in cache instead of our smarter Redis-based layer?

Reply:
Dropping these settings as Clickhouse has not implemented subquery cache usage yet:
https://github.com/ClickHouse/ClickHouse/issues/51656